deployment/flatpak/install.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Flatpak Installation
+subsection: flatpak
+section: deployment
+---
+
+# {{page.title}}
+
+## Installing Flatpak
+
+A `flatpak` package has been available for Fedora 24+ and has been installed by default on Fedora Workstation.
+
+If you do not use Fedora Workstation, you can install Flatpak by running:
+
+```
+$ sudo dnf install flatpak
+```
+
+
+
+## Installing Flatpak Builder
+
+Most applications require additional dependencies that aren’t provided by their runtimes. Flatpak allows these dependencies to be bundled as part of the application itself. In order to do this, each dependency 
```

Comment:
Casual user has to do this, or only if you want to create flatpaks yourself? If the latter one, please split this to a standalone page.

Please try to focus on best practices for Developers **on** Fedora.
See our [wiki](https://fedoraproject.org/wiki/Websites/Developer) for more info.

Reply:
Flatpak Builder is for developers who are interested in packaging their applications into flatpaks. Not for the end users.

I'll split it into another page as per your suggestion.

Reply:
`# Building Flatpacks`

_Note to self: we should probably point to that page from deployment/distribution section._